Dockerfile

```diff
@@ -1,31 +1,47 @@
+# Stage 1 build
 FROM golang:1.16.1-buster AS builder
+RUN apt-get update && apt-get install ca-certificates -y
+ENV CGO_ENABLED=0
+
+ENV USER=bubbly
+ENV UID=10001
+RUN adduser \
+    --disabled-password \
+    --gecos "" \
+    --home "/nonexistent" \
+    --shell "/sbin/nologin" \
+    --no-create-home \
+    --uid "${UID}" \
+    "${USER}"
```

Comment:
This does not look right - are you creating a new user in the _builder_ image? I suggest that the user is created in the deployment image, instead.